tools/images/ubuntu/Dockerfile

```diff
@@ -1,19 +1,15 @@
 FROM ubuntu:bionic
 MAINTAINER peter.ebden@gmail.com
 
-# Most dependencies; Go, Python, Java, Clang and Ruby (only used for fpm)
+# Go, Python, Java and other dependencies.
 RUN apt-get update && \
     apt-get install -y python3 python3-dev python3-pip openjdk-8-jdk-headless \
-    curl git locales pkg-config zlib1g-dev \
-    clang libunittest++-dev ruby ruby-dev rubygems golang-go golint && \
+    curl git locales pkg-config zlib1g-dev golang-go golint && \
```

Comment:
Looks like we need unzip here too (it maybe been pulled in transitively before)

Reply:
Yeah, weird. Maybe one of the Ruby packages needed it for some reason.